SmartGun/CORE/Logics/WarningHandler.cs

```diff
@@ -8,13 +10,12 @@ internal class WarningHandler : IWarningHandler
     {
         public void Handle(WarningMessage message)
         {
-            AddWarningToDispatcher();
+            AddWarningToDispatcher(message);
         }
 
-        private void AddWarningToDispatcher()
+        private void AddWarningToDispatcher(WarningMessage message)
         {
-            Console.WriteLine("TODO add allert to HTML here!!!");
-            //TODO add allert to HTML here!!! 
+            HTMLTableService.AddColumn(message);
```

Comment:
you add row not column rename  AddColumn -> AddRow
Ctrl + R,R use to rename method